### PR TITLE
Update 03-ViewData.Rmd

### PR DESCRIPTION
**Pull request by j-sayers**

### CODE REVIEW

File: 03-ViewData.Rmd [hunk 1]
```diff
@@ -184,13 +184,13 @@ View(RHWO)
 *Example 3*: We can further refine the Red-headed Woodpecker example (above) by filtering the species-specific data by region (e.g., [Bird Conservation Region](http://nabci-us.org/assets/images/bcr_map2.jpg) 11), time period (e.g., 2015-2019), or a combination of both.
 
 ```{r nc_species2, eval=FALSE}
-RHWO_11<-nc_count(species = 10060, region = list(bcr = "11"))
+RHWO_11 <- nc_count(species = 10060, region = list(bcr = "11"))
```
Comment (j-sayers) [inline 03-ViewData.Rmd:187]: Why do you hate spaces? (this is a single comment)

File: 03-ViewData.Rmd [hunk 2]
```diff
@@ -184,13 +184,13 @@ View(RHWO)
...
 View(RHWO_11)
 
-RHWO_year<-nc_count(species = 10060, year = c(2015, 2019))
+RHWO_year <- nc_count(species = 10060, year = c(2015, 2019))
```
Comment (j-sayers) [inline 03-ViewData.Rmd:190]: Why do you hate spaces? (this is "Start a review")

**Review by j-sayers — COMMENTED** [review]
I'm hoping you get another email from this commente

**Comment by DMEthier** [comment]
Thanks for all your wisdom.

**Pull request merged**